src/pre-engagement-form/state.ts

```diff
@@ -79,8 +79,11 @@ export const preEngagementFormReducer = (state = initialState, action: Action) =
     };
   }
 
-  if (action.type === RESET_FORM) {
-    return initialState;
+  if (action.type === RESET_FORM_STATE) {
+    return {
+      ...state,
+      formState: initialState.formState,
+    };
```

Comment:
We just needed to reset the form state here, but not the form definition. I've fixed it and renamed the constant to be more clear.